README.md

```diff
@@ -112,12 +112,15 @@ Run ClojureScript tests
 yarn test
 ```
 
-Run Cypress tests
+Run E2E tests
 
 ``` bash
-yarn e2e-test
+yarn electron-watch
+# in another shell
+yarn e2e-test # or npx playwright test
 ```
 
+
```

Comment:
Please avoid unnecessary whitespace change here😃

Reply:
Will be more careful in the future. Fixed!